### ISSUE TITLE
Add explainer on accessibility testing using XCTest

### ISSUE THREAD

**Issue by MarcSolo** [cvs-health/ios-swiftui-accessibility-techniques#1]
There are fantastic examples in this repo of using XCTest to perform accessibility related UI tests. I think it would be helpful to provide an overview in a separate markdown file that people can read before diving in to get a high level understanding of this topic.

This explainer could cover things like:

- types of issues can you check for: missing labels for elements, lack of support for Dynamic Type, asserting existence of specific semantic accessibility properties/modifiers, etc. 
- key APIs used when writing tests
- related resources like the following WWDC video: https://developer.apple.com/videos/play/wwdc2023/10035/

**Comment by pauljadam** [comment]
Great suggestion! I have some other good sources also:

https://a11y-guidelines.orange.com/en/mobile/ios/wwdc/nota11y/2023/2310035/

https://www.hackingwithswift.com/articles/148/xcode-ui-testing-cheat-sheet

https://www.polpiella.dev/xcode-15-automated-accessibility-audits

https://mobilea11y.com/guides/xcui/

**Comment by pauljadam** [comment]
.md file added to GitHub repo, thank you!!
